### DISCUSSION TITLE
Date Time Digitized

### DISCUSSION THREAD

**Discussion by tomasenskede** [photoprism/photoprism#4516] [General]
When I upload images taken with a GoPro, PhotoPrism selects the digitized date and time instead of the original date and time. Is there a way to adjust this?

This image in Adobe Lightroom has three dates
![image](https://github.com/user-attachments/assets/f4f2d1fa-3fc9-4bcc-a2a6-b88d65a9ed73)

when in PhotoPrism:
![image](https://github.com/user-attachments/assets/241457b3-8da6-4241-8e8e-b7890e7b9e78)

in windows explorer (sorry in swedish...but you understand.)
![image](https://github.com/user-attachments/assets/cd4c0cd2-01bd-4338-9f59-4879eff1871a)
![image](https://github.com/user-attachments/assets/c3a677d7-c909-4b71-99c5-6fe2dc0a336f)

PhotoPrism is the only one selecting the digitized date. Why is that, and can I change it?

**Comment by graciousgrey** [comment]
You can find an overview of the supported metadata fields here: https://www.photoprism.app/kb/metadata
As far as I can see the DateTimeOriginal should be preferred over DateTimeDigitized. 

What output do you get when running `exiftool -j your-filename.jpg` ?

**Comment by tomasenskede** [comment]
output from the file;

[{
  "SourceFile": "GOPR9281-Enhanced-NR.jpg",
  "ExifToolVersion": 12.96,
  "FileName": "GOPR9281-Enhanced-NR.jpg",
  "Directory": ".",
  "FileSize": "610 kB",
  **"FileModifyDate": "2024:09:11 15:23:42+02:00",
  "FileAccessDate": "2024:09:11 19:10:56+02:00",
  "FileCreateDate": "2024:09:11 15:10:28+02:00",**
  "FilePermissions": "-rw-rw-rw-",
  "FileType": "JPEG",
  "FileTypeExtension": "jpg",
  "MIMEType": "image/jpeg",
  "ExifByteOrder": "Little-endian (Intel, II)",
  "ImageDescription": "C:\\DCIM\\100GOPRO\\GOPR9281.GPR",
  "Make": "GoPro",
  "Model": "HERO8 Black",
  "XResolution": 240,
  "YResolution": 240,
  "ResolutionUnit": "inches",
  "Software": "Adobe Photoshop Lightroom Classic 13.5.1 (Windows)",
  "ModifyDate": "2024:09:11 15:23:42",
  "ExposureTime": "1/100",
  "FNumber": 2.8,
  "ExposureProgram": "Program AE",
  "ISO": 576,
  "ExifVersion": "0231",
  **"DateTimeOriginal": "2024:09:10 21:37:20",
  "CreateDate": "2016:03:25 15:55:23",**
  "OffsetTime": "+02:00",
  "ShutterSpeedValue": "1/100",
  "ApertureValue": 2.8,
  "ExposureCompensation": 0,
  "MaxApertureValue": 2.8,
  "MeteringMode": "Average",
  "LightSource": "Unknown",
  "Flash": "No flash function",
  "FocalLength": "3.0 mm",
  "ColorSpace": "sRGB",
  "FocalPlaneXResolution": 72,
  "FocalPlaneYResolution": 72,
  "SensingMethod": "One-chip color area",
  "FileSource": "Digital Camera",
  "SceneType": "Directly photographed",
  "ExposureMode": "Auto",
  "WhiteBalance": "Auto",
  "DigitalZoomRatio": 1,
  "FocalLengthIn35mmFormat": "15 mm",
  "SceneCaptureType": "Landscape",
  "GainControl": "None",
  "Contrast": "Normal",
  "Saturation": "Normal",
  "Sharpness": "Hard",
  "SerialNumber": "C3331350692196",
  "GPSVersionID": "2.2.0.0",
  "GPSLatitudeRef": "North",
  "GPSLongitudeRef": "East",
  "Compression": "JPEG (old-style)",
  "ThumbnailOffset": 1024,
  "ThumbnailLength": 20760,
  "DisplayedUnitsX": "inches",
  "DisplayedUnitsY": "inches",
  "CurrentIPTCDigest": "14b5bf8f9cf225732408018b35077dbc",
  "CodedCharacterSet": "UTF8",
  "ApplicationRecordVersion": 4,
  "TimeCreated": "21:37:20",
  "DigitalCreationDate": "2016:03:25",
  "DigitalCreationTime": "15:55:23",
  "Province-State": "Stockholm",
  "Country-PrimaryLocationCode": "SE",
  "Country-PrimaryLocationName": "Sweden",
  "Caption-Abstract": "C:\\DCIM\\100GOPRO\\GOPR9281.GPR",
  "PhotoshopThumbnail": "(Binary data 20760 bytes, use -b option to extract)",
  "IPTCDigest": "14b5bf8f9cf225732408018b35077dbc",
  "ProfileCMMType": "Linotronic",
  "ProfileVersion": "2.1.0",
  "ProfileClass": "Display Device Profile",
  "ColorSpaceData": "RGB ",
  "ProfileConnectionSpace": "XYZ ",
  "ProfileDateTime": "1998:02:09 06:49:00",
  "ProfileFileSignature": "acsp",
  "PrimaryPlatform": "Microsoft Corporation",
  "CMMFlags": "Not Embedded, Independent",
  "DeviceManufacturer": "Hewlett-Packard",
  "DeviceModel": "sRGB",
  "DeviceAttributes": "Reflective, Glossy, Positive, Color",
  "RenderingIntent": "Perceptual",
  "ConnectionSpaceIlluminant": "0.9642 1 0.82491",
  "ProfileCreator": "Hewlett-Packard",
  "ProfileID": 0,
  "ProfileCopyright": "Copyright (c) 1998 Hewlett-Packard Company",
  "ProfileDescription": "sRGB IEC61966-2.1",
  "MediaWhitePoint": "0.95045 1 1.08905",
  "MediaBlackPoint": "0 0 0",
  "RedMatrixColumn": "0.43607 0.22249 0.01392",
  "GreenMatrixColumn": "0.38515 0.71687 0.09708",
  "BlueMatrixColumn": "0.14307 0.06061 0.7141",
  "DeviceMfgDesc": "IEC http://www.iec.ch",
  "DeviceModelDesc": "IEC 61966-2.1 Default RGB colour space - sRGB",
  "ViewingCondDesc": "Reference Viewing Condition in IEC61966-2.1",
  "ViewingCondIlluminant": "19.6445 20.3718 16.8089",
  "ViewingCondSurround": "3.92889 4.07439 3.36179",
  "ViewingCondIlluminantType": "D50",
  "Luminance": "76.03647 80 87.12462",
  "MeasurementObserver": "CIE 1931",
  "MeasurementBacking": "0 0 0",
  "MeasurementGeometry": "Unknown",
  "MeasurementFlare": "0.999%",
  "MeasurementIlluminant": "D65",
  "Technology": "Cathode Ray Tube Display",
  "RedTRC": "(Binary data 2060 bytes, use -b option to extract)",
  "GreenTRC": "(Binary data 2060 bytes, use -b option to extract)",
  "BlueTRC": "(Binary data 2060 bytes, use -b option to extract)",
  "XMPToolkit": "Adobe XMP Core 7.0-c000 1.000000, 0000/00/00-00:00:00        ",
  "CreatorTool": "Adobe Photoshop Lightroom Classic 13.5.1 (Windows)",
  "MetadataDate": "2024:09:11 15:23:42+02:00",
  "Rating": 4,
  "Format": "image/jpeg",
  "EnhanceDetailsAlreadyApplied": true,
  "EnhanceDetailsVersion": 218300416,
  "EnhanceDenoiseAlreadyApplied": true,
  "EnhanceDenoiseVersion": 251658240,
  "EnhanceDenoiseLumaAmount": 58,
  "DateCreated": "2024:09:10 21:37:20",
  "City": "Vällingby",
  "State": "Stockholm",
  "Country": "Sweden",
  "DocumentID": "xmp.did:2a55c8e1-a139-a14b-85be-9432ee2cf74f",
  "PreservedFileName": "GOPR9281.GPR",
  "OriginalDocumentID": "F3BF4F23C4675488ADC0803910082222",
  "InstanceID": "xmp.iid:2a55c8e1-a139-a14b-85be-9432ee2cf74f",
  "Pick": 0,
  "CountryCode": "SE",
  "RawFileName": "GOPR9281-Enhanced-NR.dng",
  "Version": 16.5,
  "CompatibleVersion": 234881024,
  "ProcessVersion": 15.4,
  "ColorTemperature": 4900,
  "Tint": "+28",
  "Exposure2012": "+0.32",
  "Contrast2012": "+5",
  "Highlights2012": -57,
  "Shadows2012": "+54",
  "Whites2012": "+10",
  "Blacks2012": -17,
  "Texture": "+25",
  "Clarity2012": "+26",
  "Dehaze": 0,
  "Vibrance": "+20",
  "ParametricShadows": 0,
  "ParametricDarks": 0,
  "ParametricLights": 0,
  "ParametricHighlights": 0,
  "ParametricShadowSplit": 25,
  "ParametricMidtoneSplit": 50,
  "ParametricHighlightSplit": 75,
  "SharpenRadius": "+1.0",
  "SharpenDetail": 25,
  "SharpenEdgeMasking": 0,
  "LuminanceSmoothing": 0,
  "ColorNoiseReduction": 0,
  "HueAdjustmentRed": 0,
  "HueAdjustmentOrange": 0,
  "HueAdjustmentYellow": 0,
  "HueAdjustmentGreen": 0,
  "HueAdjustmentAqua": 0,
  "HueAdjustmentBlue": 0,
  "HueAdjustmentPurple": 0,
  "HueAdjustmentMagenta": 0,
  "SaturationAdjustmentRed": 0,
  "SaturationAdjustmentOrange": 0,
  "SaturationAdjustmentYellow": 0,
  "SaturationAdjustmentGreen": 0,
  "SaturationAdjustmentAqua": 0,
  "SaturationAdjustmentBlue": 0,
  "SaturationAdjustmentPurple": 0,
  "SaturationAdjustmentMagenta": 0,
  "LuminanceAdjustmentRed": 0,
  "LuminanceAdjustmentOrange": 0,
  "LuminanceAdjustmentYellow": 0,
  "LuminanceAdjustmentGreen": 0,
  "LuminanceAdjustmentAqua": 0,
  "LuminanceAdjustmentBlue": 0,
  "LuminanceAdjustmentPurple": 0,
  "LuminanceAdjustmentMagenta": 0,
  "SplitToningShadowHue": 0,
  "SplitToningShadowSaturation": 0,
  "SplitToningHighlightHue": 0,
  "SplitToningHighlightSaturation": 0,
  "SplitToningBalance": 0,
  "ColorGradeMidtoneHue": 0,
  "ColorGradeMidtoneSat": 0,
  "ColorGradeShadowLum": 0,
  "ColorGradeMidtoneLum": 0,
  "ColorGradeHighlightLum": 0,
  "ColorGradeBlending": 50,
  "ColorGradeGlobalHue": 0,
  "ColorGradeGlobalSat": 0,
  "ColorGradeGlobalLum": 0,
  "AutoLateralCA": 0,
  "LensProfileEnable": 0,
  "LensManualDistortionAmount": 0,
  "VignetteAmount": 0,
  "DefringePurpleAmount": 0,
  "DefringePurpleHueLo": 30,
  "DefringePurpleHueHi": 70,
  "DefringeGreenAmount": 0,
  "DefringeGreenHueLo": 40,
  "DefringeGreenHueHi": 60,
  "PerspectiveUpright": "Off",
  "PerspectiveVertical": 0,
  "PerspectiveHorizontal": 0,
  "PerspectiveRotate": 0.0,
  "PerspectiveAspect": 0,
  "PerspectiveScale": 100,
  "PerspectiveX": 0.00,
  "PerspectiveY": 0.00,
  "GrainAmount": 0,
  "PostCropVignetteAmount": 0,
  "ShadowTint": 0,
  "RedHue": 0,
  "RedSaturation": 0,
  "GreenHue": 0,
  "GreenSaturation": 0,
  "BlueHue": 0,
  "BlueSaturation": 0,
  "HDREditMode": 0,
  "ConvertToGrayscale": false,
  "OverrideLookVignette": false,
  "ToneCurveName2012": "Linear",
  "CameraProfile": "GoPro HERO8 Black",
  "CameraProfileDigest": "9275327BC560A23386A63504F7A5E1B9",
  "AutoToneDigest": "BAC5160409DE1710D52B36D9000EDB3E",
  "AutoToneDigestNoSat": "29E9D301EAFFB771328D7C022B234C76",
  "HasSettings": true,
  "CropTop": 0.148213,
  "CropLeft": 0.119682,
  "CropBottom": 0.677245,
  "CropRight": 0.88961,
  "CropAngle": -9.18,
  "CropConstrainToWarp": 0,
  "HasCrop": true,
  "AlreadyApplied": true,
  "Description": "C:\\DCIM\\100GOPRO\\GOPR9281.GPR",
  "HistoryAction": ["derived","saved","derived","saved"],
  "HistoryParameters": ["saved to new location","converted from image/dng to image/jpeg, saved to new location"],
  "HistoryInstanceID": ["xmp.iid:d7fc643d-79e3-864a-b58e-1186aedec4f3","xmp.iid:2a55c8e1-a139-a14b-85be-9432ee2cf74f"],
  "HistoryWhen": ["2024:09:11 15:06:08+02:00","2024:09:11 15:23:42+02:00"],
  "HistorySoftwareAgent": ["Adobe Photoshop Lightroom Classic 13.5.1 (Windows)","Adobe Photoshop Lightroom Classic 13.5.1 (Windows)"],
  "HistoryChanged": ["/","/"],
  "DerivedFromInstanceID": "xmp.iid:d7fc643d-79e3-864a-b58e-1186aedec4f3",
  "DerivedFromDocumentID": "xmp.did:d7fc643d-79e3-864a-b58e-1186aedec4f3",
  "DerivedFromOriginalDocumentID": "F3BF4F23C4675488ADC0803910082222",
  "ToneCurvePV2012": ["0, 0","255, 255"],
  "ToneCurvePV2012Red": ["0, 0","255, 255"],
  "ToneCurvePV2012Green": ["0, 0","255, 255"],
  "ToneCurvePV2012Blue": ["0, 0","255, 255"],
  "PointColors": "-1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000",
  "MaskGroupBasedCorrWhat": "Correction",
  "MaskGroupBasedCorrAmount": 1,
  "MaskGroupBasedCorrActive": true,
  "MaskGroupBasedCorrCorrectionName": "Mask 1",
  "MaskGroupBasedCorrCorrectionSyncID": "990DED27CB5A8D41B50A93916AAB4D9B",
  "MaskGroupBasedCorrExposure": 0,
  "MaskGroupBasedCorrHue": 0,
  "MaskGroupBasedCorrSaturation": 0,
  "MaskGroupBasedCorrContrast": 0,
  "MaskGroupBasedCorrClarity": 0,
  "MaskGroupBasedCorrSharpness": 0,
  "MaskGroupBasedCorrBrightness": 0,
  "MaskGroupBasedCorrToningHue": 0,
  "MaskGroupBasedCorrToningSaturation": 0,
  "MaskGroupBasedCorrExposure2012": -0.175,
  "MaskGroupBasedCorrContrast2012": 0,
  "MaskGroupBasedCorrHighlights2012": 0,
  "MaskGroupBasedCorrShadows2012": 0,
  "MaskGroupBasedCorrWhites2012": 0,
  "MaskGroupBasedCorrBlacks2012": 0,
  "MaskGroupBasedCorrClarity2012": 0,
  "MaskGroupBasedCorrDehaze": 0,
  "MaskGroupBasedCorrLuminanceNoise": 0,
  "MaskGroupBasedCorrMoire": 0,
  "MaskGroupBasedCorrDefringe": 0,
  "MaskGroupBasedCorrTemperature": 0,
  "MaskGroupBasedCorrTint": 0,
  "MaskGroupBasedCorrTexture": 0,
  "MaskGroupBasedCorrLocalGrain": 0,
  "MaskGroupBasedCorrLocalCurveRefineSaturation": 100,
  "MaskGroupBasedCorrMaskWhat": "Mask/Gradient",
  "MaskGroupBasedCorrMaskMaskActive": true,
  "MaskGroupBasedCorrMaskMaskName": "Linear Gradient 1",
  "MaskGroupBasedCorrMaskMaskBlendMode": 0,
  "MaskGroupBasedCorrMaskMaskInverted": false,
  "MaskGroupBasedCorrMaskMaskSyncID": "1C1357B0AB78AC47A702076C8AE00388",
  "MaskGroupBasedCorrMaskValue": 1,
  "MaskGroupBasedCorrMaskZeroX": 0.180426,
  "MaskGroupBasedCorrMaskZeroY": 0.434675,
  "MaskGroupBasedCorrMaskFullX": 0.70332,
  "MaskGroupBasedCorrMaskFullY": 0.330678,
  "DCTEncodeVersion": 100,
  "APP14Flags0": "[14], Encoded with Blend=1 downsampling",
  "APP14Flags1": "(none)",
  "ColorTransform": "YCbCr",
  "ImageWidth": 2787,
  "ImageHeight": 2058,
  "EncodingProcess": "Baseline DCT, Huffman coding",
  "BitsPerSample": 8,
  "ColorComponents": 3,
  "YCbCrSubSampling": "YCbCr4:4:4 (1 1)",
  "Aperture": 2.8,
  "ImageSize": "2787x2058",
  "Megapixels": 5.7,
  "ScaleFactor35efl": 5.0,
  "ShutterSpeed": "1/100",
  "SubSecModifyDate": "2024:09:11 15:23:42+02:00",
  "ThumbnailImage": "(Binary data 20760 bytes, use -b option to extract)",
  "GPSLatitude": "59 deg 21' 54.00\" N",
  "GPSLongitude": "17 deg 52' 4.80\" E",
  "DateTimeCreated": "2024:09:10 21:37:20",
  "DigitalCreationDateTime": "2016:03:25 15:55:23",
  "CircleOfConfusion": "0.006 mm",
  "FOV": "100.4 deg",
  "FocalLength35efl": "3.0 mm (35 mm equivalent: 15.0 mm)",
  "GPSPosition": "59 deg 21' 54.00\" N, 17 deg 52' 4.80\" E",
  "HyperfocalDistance": "0.53 m",
  "LightValue": 7.1
}]

**Reply by lastzero** [reply]
I don't see the word “Digitized” anywhere here? The problem is that `CreateDate` just seems to be wrong. How can the image be created before it was taken? If anything, `CreateDate` should be after `DateTimeOriginal`, but in general - from what I've read here and what I remember - `CreateDate` is a field used for videos, not photos:

- https://exiftool.org/forum/index.php?topic=9411.0
- https://exiftool.org/forum/index.php?topic=2568.0

However, it is true that PhotoPrism favors certain fields over others, e.g. if they have a higher time resolution or contain a time zone:
- https://github.com/photoprism/photoprism/blob/develop/internal/meta/data.go#L22C2-L24

**Reply by lastzero** [reply]
Can you please run `exiftool -j -g -n filename.jpg` to see exactly where the data is coming from? Thank you!
